test/sql/updates/post.v9.sql

```diff
@@ -0,0 +1,5 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+--\ir post.v8.sql
```

Comment:
Why this is commented?? Shouldn't include the post.v8 and then execute the post.v9 stuff??

Reply:
Nit: missing new line at the end of file!

Reply:
I will remove this file, we do not this file anymore, since we already have post.continuous_aggs.v9, right...?

Reply:
This is wrong @RafiaSabih you should include the post.v8.sql and post.continuous_aggs.v9.sql. To summarize the content of this file should be
```sql
\ir post.v8.sql
\ir post.continuous_aggs.v9.sql
```